dozer/cogs/_utils.py

```diff
@@ -1,13 +1,18 @@
 """Utilities for Dozer."""
 import asyncio
 import inspect
+import logging
 import typing
 from collections.abc import Mapping
 
 import discord
 from discord.ext import commands
 
-__all__ = ['bot_has_permissions', 'command', 'group', 'Cog', 'Reactor', 'Paginator', 'paginate', 'chunk', 'dev_check']
+from dozer import db
+
+__all__ = ['bot_has_permissions', 'command', 'group', 'Cog', 'Reactor', 'Paginator', 'paginate', 'chunk', 'dev_check', 'DynamicPrefixEntry']
+
+logger = logging.getLogger("dozer")
```

Comment:
Bit nit-picky, but let's please call this `DOZER_LOGGER` like we have in the rest of the bot, for consistency's sake